src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -1896,8 +1946,11 @@ public new View Parent
 
                 if (basehandle is Layer layer)
                 {
-                    ret = new View(Layer.getCPtr(layer).Handle, false);
-                    NUILog.Error("This Parent property is deprecated, shoud do not be used");
+                    using (View view = new View(Layer.getCPtr(layer).Handle, false))
+                    {
+                        ret = view;
```

Comment:
This does not make sense.
When it gets out of the `using` scope, the`ret` value will be disposed.
It need to be fixed.